vector/src/main/scala/geotrellis/vector/triangulation/DelaunayTriangulation.scala

```diff
@@ -779,8 +789,8 @@ case class DelaunayTriangulation(pointSet: DelaunayPointSet, halfEdgeTable: Half
 }
 
 object DelaunayTriangulation {
-  def apply(pointSet: DelaunayPointSet, debug: Boolean = false) = {
+  def apply(pointSet: DelaunayPointSet, tolerance: Double = 1e-8, debug: Boolean = false) = {
```

Comment:
This change caused `geotrellis.pointcloud.spark.triangulation.BoundaryDelaunaySpec.scala` 
and `geotrellis.vector.triangulation.DelaunayTriangulationSpec.scala` specs fail. 

Change `DelaunayTriangulation(pts, false)` to `DelaunayTriangulation(pts, debug = false)`